inject-dist/observers.js

```diff
@@ -16,24 +17,67 @@ const OBSERVERS = [
   'navigator.__proto__.*',
   'performance.__proto__.*',
   'performance.timing.__proto__.*',
+  'Math.__proto__.*',
   'Array.prototype.*',
   'String.prototype.*',
   'Object.prototype.*',
   'CSSStyleDeclaration.prototype.*',
+  'MimeType.prototype.*',
+  'Plugin.prototype.*',
+  'Screen.prototype.*',
   'document.featurePolicy',
   'document.write',
   'queueMicrotask',
   'requestIdleCallback',
-  'navigator.scheduling.isInputPending',
   'scheduler.postTask',
+  'matchMedia',
+  'navigator.scheduling.isInputPending',
+  'OfflineAudioContext',
+  'OfflineAudioContext.prototype.createOscillator',
+  'OfflineAudioContext.prototype.createDynamicsCompressor',
+  'HTMLCanvasElement.prototype.getContext',
+  'CanvasRenderingContext2D.prototype.rect',
+  'CanvasRenderingContext2D.prototype.fillRect',
+  'CanvasRenderingContext2D.prototype.fillText',
+  'CanvasRenderingContext2D.prototype.beginPath',
+  'CanvasRenderingContext2D.prototype.arc',
+  'CanvasRenderingContext2D.prototype.fill',
+  'HTMLCanvasElement.prototype.toDataURL',
+  'screen.colorDepth',
+  'DOMRect.prototype.width',
+  'DOMRect.prototype.height',
+  'HTMLSpanElement.prototype.offsetHeight',
+  'HTMLSpanElement.prototype.offsetWidth',
+  'indexedDB',
+  'openDatabase',
+  'DateTimeFormat.prototype.resolvedOptions',
+  'chrome',
+  'navigator.userAgent',
```

Comment:
I think you can remove this as it's covered by `'navigator.__proto__.*'` above. That might be why it was failing in your test.

Reply:
Apparently FingerprintJS doesn't request `navigator.userAgent`, so probably that's why there were no results. There still appear some missing property-accesses, so will still look into those.

Reply:
Still, I think it'd be good to remove the redundancy. I'll suggest and commit.

```suggestion
```